src/importexport/guitarpro/internal/gtp/gp67dombuilder.cpp

```diff
@@ -558,8 +558,8 @@ std::pair<int, std::shared_ptr<GPBeat> > GP67DomBuilder::createGPBeat(XmlDomNode
     auto hairpinType = [](const String& str) {
         if (str == u"Crescendo") {
             return GPBeat::Hairpin::Crescendo;
-        } else if (str == u"Decrescendo") {
-            return GPBeat::Hairpin::Decrescendo;
+        } else if (str == u"Decrescendo") { // this is what GTP calls it internally
```

Comment:
Comments like this don't seem valuable to me beyond the context of this PR, so I don't think we should include them in the code (otherwise we could add such comments to almost every string that appears in import code)